content/en/guides/employee-engagement.md

```diff
@@ -2,198 +2,201 @@
 
-### Suggested tactics for starting early
+### Tips for starting early
 
-* Before you have code, join the discussion, track changes, and listen.
+* Before you have written any code, join the discussion, track changes, and listen.
 * Ask questions.
 * Pick up the easy work. Start contributing to community needs already identified.
 * Always be transparent about where you come from, i.e. the organization you are working for.
 * When you first join a community, don't be afraid to acknowledge those who contributed before you.
-* Starting early means talking/contributing before things are perfect. Don't be afraid to show yourself. Use levity and be a human -- no one is a widget in a community -- but be careful with your language and humor. These people don't know you yet, and you are probably not that funny anyway. ;)
-* This is a geographically diverse and most likely international space. When you start early, listen and be mindful of differences, e.g language differences, racial differences, pay differences.
-* Show your work, show your process, show your thinking -- before it's done.
-* Be excited, you're here early, for a reason.
+* Starting early means talking and contributing before things are perfect. Don't be afraid to show yourself. Be light-hearted and be a human - no one is a widget in a community. Do be mindful of your choice of language and humor. When people don't know you, you may be misunderstood and this can cause extra problems.
+* Be aware that you are probably in an international and geographically diverse space. Listen and be mindful of differences, e.g language differences, cultural differences, pay differences.
+* Show your work, show your process, show your thinking - before it's done.
+* Be excited, you're here for a reason!
 
 
-## Principle 2: Puts the Community First.
+## Principle 2: Put the Community First.
 
-*Balance interests and seek alignment. If in doubt put the interests of the community first. This will have the better long-term results. Engage as individual but keep your affiliation and the interests of your organization in mind. Never forget that open source is a collaborative effort and that the collective holds the timeline.*
+*Balance the interests of everyone and seek alignment. If you are not sure, put the interests of the community first. This will have the better long-term results. Engage as an individual but remember that you represent your organization and its interests. Never forget that open source is a collaborative effort and that the collective holds the timeline.*
```

Comment:
I would avoid talking about representing your organization here, because contributors most often are not formal representatives of their organization. But they do have to keep in mind that they have an obligation to their employer as well as to the project they are contributing to.

Reply:
Thanks! Glad to have the nuance picked up. I added back the original sentence.